frontend/src/components/mainView/ViewHeader.js

```diff
@@ -77,7 +77,7 @@ const ViewHeader = (props) => {
 			setState(dispatch, {currentViewName: 'Card', listView: true});
 		}
 	},[dispatch])
-
+    //if searchSettings.isFilterUpdate
```

Comment:
If you don't need this take it out. Less clutter the better.

Reply:
Removed it